doc-content/enterprise-only/employee-rostering/er-rotation-edit-proc.adoc

```diff
@@ -1,17 +1,30 @@
 [id='er-rotation-edit-proc']
 = Viewing and editing the rotation pattern
+The rotation pattern enables you to add, move, and delete shifts so you can manage your employee resources efficiently.
 
-The rotation pattern contains shift templates that are automatically replicated for every period. For details about the rotation pattern, see <<er-rotation-con>>.
+The rotation pattern is defined by time buckets and seats. A time bucket describes a particular time slot, for example  9:00 a.m. to 5:00 p.m., for a particular spot, for example Airport South, and any skills that are required, for example firearm training.
 
-You can view the rotation pattern, add, move, and delete shifts.
+A seat is an employee assignment for a particular day in a specific time bucket.
+
+A stub list is a list of employees available to be assigned to a time bucket.
+
+For more information about the rotation pattern, see <<er-rotation-con>>.
 
 .Procedure
 . To view and edit the rotation pattern in the Employee Rostering application web interface, click the *Rotation* tab.
-. To view additional days of the rotation pattern, use the horizontal scroll bar at the bottom of the browser window.
-. To add a shift template, click an empty area of the table. The application adds a shift, determining the slot and time span automatically from the location of the click.
-. To edit a shift template, click the shift template. You can set the following values:
-** *From* and *To* day and time: The exact time and duration of the shift. The day is counted from the start of the rotation period.
-** *Rotation employee*: The employee assigned to the shift by default. Select *Unassigned* if you do not want to assign any employee to this shift by default; in this case, the rostering process determines the employee in the shift without a preferred value.
+. Select a spot form the *Rotation* menu.
+. To add a time bucket, click *Add New Time Bucket*. The *Creating Working Time Bucket* dialog appears.
+. Specify a start and end time, select any additional required skills, select the days for this time bucket, and click *Save*. The time bucket appears on the *Rotation* page.
+. To create an employee stub list to add employees to the rotation, click *Edit Employee Stub List*.
+. In the *Edit Employee Stub List* dialog, click *Add Employee* and select an employee from the list.
+. Add all of the employees required for this stub list and click *Save*. The employees appear above the time buckets on the *Rotation* page.
+. Drag an employee to a time bucket. The time bucket is populated with the employee icon.
```

Comment:
I think ". Select an employee from the employee stub list by clicking on the employee icon", ". Clicking and dragging your mouse over the slots/seats of a time bucket assign the currently selected employee to that particular seat/slot of the time bucket." might be a little clearer.